.circleci/config.yml

```diff
@@ -12,44 +12,36 @@ commands:
   install_macos_dependencies:
     steps:
       - run: brew install protobuf cmake ccache libtool libspatialite pkg-config luajit curl wget czmq lz4 spatialite-tools unzip
-      - run: pip3 install requests shapely conan
+      - run: /usr/bin/python3 -m pip install --user requests
+      - run: pip3 install shapely conan
       - run: git clone https://github.com/kevinkreiser/prime_server --recurse-submodules && cd prime_server && ./autogen.sh && ./configure && make -j8 && make install
+  install_linux_dependencies:
+    steps:
+      - run: sudo bash ./scripts/install-linux-deps.sh
 
 jobs:
   build-docker-images:
-    docker:
-      - image: docker:18.06.3-ce-git
+    machine:
+      image: ubuntu-2004:current
```

Comment:
this should have a current docker version installed: https://circleci.com/developer/machine/image/ubuntu-2004#how-this-image-works. also `build-essentials` and others, so installing deps will take less time